src/backend/dx12/src/command.rs

```diff
@@ -519,11 +535,23 @@ impl CommandBuffer {
     }
 
     fn set_graphics_bind_point(&mut self) {
-        if self.active_bindpoint != BindPoint::Graphics {
-            // Switch to graphics bind point
-            let (pipeline, _) = self.gr_pipeline.pipeline.expect("No graphics pipeline bound");
-            unsafe { self.raw.SetPipelineState(pipeline); }
-            self.active_bindpoint = BindPoint::Graphics;
+        match self.active_bindpoint {
+            BindPoint::Graphics => { }, // Nothing to do
+            BindPoint::Compute => {
+                // Switch to graphics bind point
+                let (pipeline, _) = self.gr_pipeline.pipeline.expect("No graphics pipeline bound");
+                unsafe { self.raw.SetPipelineState(pipeline); }
+                self.active_bindpoint = BindPoint::Graphics;
+            },
+            BindPoint::InternalGraphics => {
```

Comment:
I wonder if we should just have a flag inside `BindPoint::Graphics { internal_signature: bool }` for merging it with the internal one